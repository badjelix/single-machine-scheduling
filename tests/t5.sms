50
10 17 27 2 9 8
7 18 61 4 1 5 7 5
4 14 32 4 3 5 3 3
2 2 8 2 1 1
21 15 66 5 4 4 2 2 3
25 1 26 1 1
15 3 18 3 1 1 1
15 16 63 2 11 5
18 17 35 1 17
13 16 61 5 3 4 4 1 4
21 17 72 4 4 3 4 6
1 17 52 5 3 5 1 4 4
10 2 16 2 1 1
9 9 27 1 9
8 5 23 5 1 1 1 1 1
0 6 18 1 6
14 8 38 3 5 1 2
5 8 21 6 1 1 3 1 1 1
18 17 52 1 17
10 3 16 3 1 1 1
10 7 31 2 3 4
15 13 54 4 3 4 4 2
5 17 56 1 17
7 16 39 3 4 6 6
29 16 61 1 16
17 6 35 5 1 1 2 1 1
19 4 31 3 2 1 1
20 13 33 2 4 9
22 7 43 3 3 3 1
27 6 45 2 5 1
29 9 47 4 2 1 4 2
16 3 25 1 3
24 10 44 1 10
21 5 36 5 1 1 1 1 1
10 14 52 2 9 5
10 13 23 2 8 5
10 13 23 2 7 6
10 12 46 5 3 3 3 2 1
11 5 16 3 3 1 1
20 9 38 3 2 3 4
27 18 81 1 18
8 2 14 2 1 1
21 11 43 6 2 1 1 2 2 3
12 12 24 4 3 4 4 1
19 13 32 4 2 6 3 2
3 5 13 5 1 1 1 1 1
20 3 23 3 1 1 1
1 5 11 1 5
28 2 30 2 1 1
0 11 11 1 11
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
