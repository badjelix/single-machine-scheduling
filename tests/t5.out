14
4 6 7
6 25
7 15 16 17
13 12 14
20 10 11 13
25 45
27 26 28 29
29 22 35 38
30 39 44
32 18
34 21 30 31 32 33
41 62
42 8 9
48 1
